bar/bar.cc

```diff
@@ -44,25 +44,7 @@ int main(int argc, char *argv[]) {
     gettimeofday(&tp, NULL);
     long int start_ms = tp.tv_sec * 1000 + tp.tv_usec / 1000;
 
-    pid_t pid = getpid();
-    std::string mypid = std::to_string(pid);
-
-    std::string pid_file = "/var/run/user/" + std::to_string(getuid()) + "/nwgbar.pid";
-
```

Comment:
@nwg-piotr I guess the code must have changed at some point, since this implementation I am removing currently puts it into `/var/run/user/` ...

Reply:
Well, I need to pay more attention to details before merging.

Reply:
I will get something going for `/tmp`, then.

Reply:
I believe you changed it here: c0c86186272daad251d30e114986f5c89cf18965